src/components/layouts/register/layout.styles.tsx

```diff
@@ -0,0 +1,79 @@
+import { gridBreakpoint } from "@/shared/breakpoints";
+import styled from "styled-components";
+
+export const StylesWrapper = styled.div<{}>`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+
+  .image-side {
+    position: relative;
+    width: 100vw;
+
+    div {
```

Comment:
🦆 in general we want to avoid setting html attributes directly into a `tag`, this is because tags hold very little specificity compared to a class, also, if the HTML changes, this will automatically fail. You can follow a convention, to further add classes to your component HTML :)

Reply:
🥔 I have changed the `tag` with `className` instead, and updated the style